project/CodeCoverageSettings.scala

```diff
@@ -4,8 +4,10 @@ import scoverage.ScoverageKeys
 object CodeCoverageSettings {
 
   val settings: Seq[Setting[_]] = Seq(
-    ScoverageKeys.coverageMinimumStmtTotal := 70,
+    ScoverageKeys.coverageMinimumStmtTotal := 100,
     ScoverageKeys.coverageFailOnMinimum := true,
-    ScoverageKeys.coverageHighlighting := true
+    ScoverageKeys.coverageHighlighting := true,
+    ScoverageKeys.coverageExcludedFiles := ".*\\/Routes\\/Module;.*" ,
```

Comment:
this needs to be double checked, I don't think coverage is working any more, the coverage report is not visible any more